readme.md

```diff
@@ -3,34 +3,19 @@ Repository for both **NEST** and **Elasticsearch.Net**, the two official [Elasti
 ## Compatibility Matrix
 <table>
     <tr>
+        <th><b>.NET Clients<b></th>
         <th><b>Elasticsearch<b></th>
-        <th><b>Clients<b></th>
         <th><b>Supported<b></th>
         <th><b>Windows/Linux CI</b></th>
         <th><b>Tests<b></th>
     </tr>
     <tr>
-    	<td><code>0.x</code></td>
-    	<td><code>0.x</code></td>
+    	<td><code>0.x - 2.x</code></td>
+    	<td><code>0.x - 2.x</code></td>
```

Comment:
I would keep these as separate major versions, to illustrate that 1.x client -> 1.x ES, etc.

Reply:
I think its fairly obvious from the rest of the table?

Reply:
I appreciate what you're saying, but is there harm in being completely explicit, aside from the extra table rows, since the aim is to provide guidance for other version -> version scenarios?